mtp_bank_admin/apps/bank_admin/tests/test_adi.py

```diff
@@ -308,3 +308,12 @@ def test_accounting_date_is_set_back_across_month_boundary(self, mock_date, mock
                 receipt_date.strftime(adi_config.ADI_DATE_FORMAT),
                 journal_ws[adi_config.ADI_DATE_CELL].value
             )
+
+    def test_early_reconciliation_raises_error(self, mock_api_client):
+        try:
+            self._generate_test_adi_journal(
```

Comment:
you can do `with self.assertRaises(EarlyReconciliationError):` which makes this a little cleaner, but no big deal either way :)